Method: ExceptionTryCatchFinallyWhen.Program.<Main>g__Throw|0_0[void..()]

Trace 0
	[start] ExceptionTryCatchFinallyWhen.Program.<Main>g__Throw|0_0[void..()]
	[ end ] ExceptionTryCatchFinallyWhen.Program.<Main>g__Throw|0_0[void..()]

Method: ExceptionTryCatchFinallyWhen.Program.Main[void..(class.System.String[])]

Trace 0
	[start] ExceptionTryCatchFinallyWhen.Program.Main[void..(class.System.String[])]
		[start] ExceptionTryCatchFinallyWhen.Program.<Main>g__Throw|0_0[void..()]
		[ end ] ExceptionTryCatchFinallyWhen.Program.<Main>g__Throw|0_0[void..()]
	[ end ] ExceptionTryCatchFinallyWhen.Program.Main[void..(class.System.String[])]

